variables.tf

```diff
@@ -7,6 +7,16 @@ variable "deployment_id" {
   }
 }
 
+variable "aws_region" {
+  description = "AWS Region to be used"
+  type = string
+
+  validation {
+  condition = can(regex("[a-z][a-z]-[a-z]+-[1-9]", var.aws_region))
+  error_message = "Must be valid AWS Region names."
```

Comment:
Since just one region needed, change to "Must be a valid AWS Region name."

The other question is if we should default this to us-east-1?

Reply:
Yes I think we should have default region to us-east-1